FAQ.md

```diff
@@ -31,35 +31,35 @@ The main reason it works that way is because there are two main types of ddos:
 
 - Actual denial of service that aims to remain as undetected as possible by simulating plausible traffic and hitting concrete vulnerabilities in the target (or targets). This type of ddos doesn't require a lot of traffic and thus is mostly limited by the amount of clients generating this type of load (or rather unique IPs)
 
-## Should I care about costs if I run an app in public cloud?
+**Should I care about costs if I run an app in public cloud?**
 
 TL;DR: [yes](https://github.com/Arriven/db1000n/issues/153)
 
 Cloud providers could charge a huge amount of money not only for compute resources but for traffic as well.
 If you run an app in the cloud please control your billing (if you use Docker, ensure that use advanced image: `ghcr.io/arriven/db1000n-advanced`)
 
-## Can I leave the site for the night?
+**Can I leave the site for the night?**
 
 Yes, you can. I personally leave the browser on overnight and it works fine.
 
-### How can I make sure that the computer does not go to sleep while the site is running?
+**How can I make sure that the computer does not go to sleep while the site is running?**
 
 To do this, you need to install a program which keeps the screen turned off. Instructions for different operating systems below:
 
-#### I have Windows
+**_I have Windows:_**
 
 Program name: **Caffeinated ([download](https://www.microsoft.com/en-us/p/windows-caffeinated/9pbvhhsn78bl?activetab=pivot:overviewtab))**
 
-#### I have Mac OS
```

Comment:
I guess, this was kinda OK - https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#headings

Reply:
it was too big of a change in headings and it triggered another markdown violation. Updated it to be a list instead